components/CanvasOne.js

```diff
@@ -0,0 +1,82 @@
+import { useRef, useEffect, useState } from "react";
+import styled from "styled-components";
+
+function lineWidthRandomizer() {
+  const randomNumber = Math.random();
+  if (randomNumber <= 0.4) {
+    return 2;
+  } else if (randomNumber <= 0.8) {
+    return 8;
+  } else {
+    return 4;
+  }
+}
+
+let firstLineWidth = lineWidthRandomizer();
+let secondLineWidth = lineWidthRandomizer();
+let thirdLineWidth = lineWidthRandomizer();
+let fourthLineWidth = lineWidthRandomizer();
+
+export default function CanvasOne() {
+  const canvasRef = useRef(null);
+  const [backgroundColor, setBackgroundColor] = useState(148);
+  useEffect(() => {
+    const canvas = canvasRef.current;
+    const ctx = canvas.getContext("2d");
```

Comment:
to understand your code a bit better you should name your variables like "canvasTextContent", or like your line 15, 16, 17 or 18. We think you've done this because you use this variable a lot, but in the future it will be difficult to understand what ctx is.

Reply:
Hi Jonas, ctx is the short version of context. It is quite common to use it but you are 100% right much better to avoid abbreviations! I changed it :)